Composer/packages/server/src/models/bot/feedManager.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as msRest from '@azure/ms-rest-js';
+
+import logger from './../../logger';
+
+const log = logger.extend('feed-manager');
+
+// http client for fetch json feed from url
+const clientOptions: msRest.ServiceClientOptions = {
+  requestPolicyFactories: [msRest.logPolicy(log)],
+};
+const token = process.env.ACCESS_TOKEN || 'token';
+const creds = new msRest.TokenCredentials(token);
+const client = new msRest.ServiceClient(creds, clientOptions);
+
+// TODO: update feed url to main
+const feedUrl = 'https://raw.githubusercontent.com/microsoft/BotFramework-Composer/zhixzhan/homepage-refine/feeds.json';
```

Comment:
need update this url later

Reply:
One client high frequency fetch this url,  seems will be forbidden by Github for some  RPS policy I guess. will meet an error
```
{"message":"request to https://raw.githubusercontent.com/microsoft/BotFramework-Composer/zhixzhan/homepage-refine/feeds.json failed, reason: connect ECONNREFUSED 0.0.0.0:443"}
```

In my development environment, after 50+ times fetch in one day, I will meet this error.